dil/pom.xml

```diff
@@ -48,6 +48,36 @@
 		  <artifactId>utils</artifactId>
 		  <version>${assimbly.version}</version>
 	  </dependency>
+
```

Comment:
Two remarks:

- For Assimbly the scope cannot be 'provided'. In Karaf this is used because the dependencies are injected by Karaf into runtime. In Assimbly the dependencies are provided in compile time.
- Maybe it's good to check if these dependencies aren't already provided by the base repository or add them there. The reason is that dependencies can be managed centrally in one list in the base.

Reply:
👍🏻